qiskit/pulse/instructions/directives.py

```diff
@@ -55,3 +55,80 @@ def channels(self) -> Tuple[chans.Channel]:
     def __eq__(self, other):
         """Verify two barriers are equivalent."""
         return isinstance(other, type(self)) and set(self.channels) == set(other.channels)
+
+
+class AreaBarrier(Directive):
+    """Pulse ``AreaBarrier`` directive.
+
+    This instruction is intended to be used internally within the pulse builder,
+    to naively convert :class:`.Schedule` into :class:`.ScheduleBlock`.
```

Comment:
```suggestion
    to convert :class:`.Schedule` into :class:`.ScheduleBlock`.
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/8949/commits/7ab2710f05605a1d70bbd3d5405243092d0340c1